python/shotgun_model/shotgun_hierarchy_model.py

```diff
@@ -119,8 +131,141 @@ def item_from_path(self, path):
             :meth:`~shotgun-api3:shotgun_api3.Shotgun.nav_expand()` method.
         :returns: :class:`~PySide.QtGui.QStandardItem` or ``None`` if not found
         """
-        self._log_debug("Resolving model item for path %s" % path)
-        return self._ensure_item_loaded(path)
+        self._logger.debug("Resolving model item for path %s" % path)
```

Comment:
just `logger.debug("xxx")` once you have moved it to the top of the file. That's the recommended pattern these days.